organization/affiliates.md

```diff
@@ -0,0 +1,15 @@
+---
+type: resource
+---
+
+# Affiliates
+
+The Foundation for Public Code is a member-owned association that exists in an ecosystem of affiliates and [partners](partnerships.md). Our affiliates help us to [achieve our mission](https://about.publiccode.net/organization/mission.html). Read more about [becoming an affiliate](../activities/creating-affiliations/index.md).
```

Comment:
The definition makes it sound much more like an equal partnership (or quid pro quo) - this sentence sounds quite hierarchical.
```suggestion
The Foundation for Public Code is a member-owned association that exists in an ecosystem of affiliates and [partners](partnerships.md). Affiliates are non-profit organizations whose work aligns with ours and that help us to [achieve our mission](https://about.publiccode.net/organization/mission.html). Read more about [becoming an affiliate](../activities/creating-affiliations/index.md).
```